packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -665,9 +688,34 @@ export const GiantTrackTile = ({
       >
         <TrackMetadataList trackId={trackId} />
         {description ? (
-          <UserGeneratedText tag='h3' size='s' lineHeight='multi'>
-            {description}
-          </UserGeneratedText>
+          <Flex column gap='m'>
+            <Flex ref={descriptionRef} column>
+              <UserGeneratedText
+                tag='h3'
+                size='s'
+                lineHeight='multi'
+                css={{
+                  display: '-webkit-box',
```

Comment:
We need this?

Reply:
it's like similar to `display: box` but slightly different

Reply:
actually nvm the way in left nav is better